docs/versioned_docs/version-5.18/configuration/images/rebuild-strategy.mdx

```diff
@@ -19,9 +19,9 @@ DevSpace will always rebuild when the `-b / --force-rebuild` flag is explicitly
 :::
 
 #### Options For `rebuildStrategy`
-Currently DevSpace supports the rebuild strategies `always` and `ignoreContextChanges`:
-- **always**: if this strategy is provided, DevSpace will always rebuild the image
-- **ignoreContextChanges**: same as default except that DevSpace will ignore changes to files within the docker context
+Currently DevSpace supports two rebuild strategies:
+- `always` if this strategy is provided, DevSpace will always rebuild the image
+- `ignoreContextChanges` same as default except that DevSpace will ignore changes to files within the docker context
```

Comment:
@FabianKramm which one is the default here? and what does it mean by `same as default`

Reply:
@LukasGentele actually there is a third option which is the default which is used when not specifying this option which is like `ignoreContextChanges`, but will rebuild when there were changes to files within the docker context